test/unit/pdf_find_controller_spec.js

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2018 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { buildGetDocumentParams } from './test_utils';
+import { EventBus } from '../../web/ui_utils';
+import { getDocument } from '../../src/display/api';
+import { PDFFindController } from '../../web/pdf_find_controller';
+
+class MockLinkService {
```

Comment:
Indeed, that's why I chose for a mock here. I initially used the `SimpleLinkService` class, but quickly realized that it was a bit too simple for the testing purposes.

Reply:
One more tiny nit here: How about importing `SimpleLinkService` above and do `class MockLinkService extends SimpleLinkService` here, since that will first of all guarantee that all of the expected methods are present and second of all will also help "document" the `MockLinkService` w.r.t. the expected interface!?